modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

```diff
@@ -0,0 +1,32 @@
+= Manage API Group Instance Contracts
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After you grant access to API Group instances in Exchange, a contract is automatically created for it. You can then manage these contracts from API Manager.
+
+To view API Group instance contracts:
+
+. In API Manager, click *API Groups*.
+. From the API Group instance details page, click the `version:instance` link for the API Group instance.
++
+The API Group instance details page is displayed.
+. From the left navigation, click *Contracts* and expand the instance for which you want to view the contract information:
++
+image:api-group-instance-contracts.png[View Contracts for API Group Instance]
++
+The contracts for the API Group instances are displayed.
+
+
+To manage API Group instance contracts:
+
+. In API Manager, click *API Groups*.
+. From the API Group instance details page, click the `version:instance` link for the API Group instance.
++
+The API Group instance details page is displayed.
+. From the left navigation, click *Contracts* and expand the instance for which you want to view the contract information:
++
+image:ap-group-instance-manage-contracts.png[Manage Contracts for API Group Instance]
++
+The contracts for the API Group instances are displayed.
+. For the API Group instance for which you want to manage the contract, click the ellipses button and click *Revoke*.
```

Comment:
I think we should add information about the new API contracts page because with API groups in every API contracts page you will see the API contracts and the Group contracts of that API. Also, we need to introduce the concept of `Active` contract and show the users how they will see it in the UI.

Reply:
Done! Thanks!

Reply:
Sorry to reopen this conversation, but please add a section to add this `we need to introduce the concept of Applied contract and show the users how they will see it in the UI.`

Reply:
## Why we are introducing applied contracts? 
### Before API groups
The consumer has an application APP and he requests access to API A, so a contract is created between APP and A. There is no way to create another contract with the application APP and API A.

### With API groups
The consumer has an application APP and he requests access to API A, so a contract is created between APP and A. Then, consumer use the same application APP, to request access a group instance G **which has API A in it**, so a contract is created between APP and G.
So now, APP has two contracts:
- One directly to API A
- One with group instance G, so indirectly it has another contract with API A through G. 
As can be seen, APP has two contracts with API A, one directly and one though group instance G.
Which contract will be applied? the contract made directly to the API or the contract made through the group instance G?
The one will be applied we call it `applied` contract and it can be seen in the UI.

## How we select the applied contract?
We mark a contract as applied for an API and an application when it is has the **less restrictive** tier when comparing with the other tiers of the contracts between that APP and that API. Of course, only `approved` contracts are eligible to be `applied` (a revoked contract cannot be the applied one because it is not active)

For example: Let's use the example of the last section:
API contract tier: `100 requests per minute`
Group contract: `10 requests per minute`
The applied one will be the API contract because it is the less restrictive one, it gives the user 90 requests more per minute than the group contract.

Please keep in mind that a contract is applied (or not) for an API, no for a group. For example, using the same example, let's suppose the consumer also creates a contract between the same group instance G and another application (let's call it APP 2).
 - For APP 1 and API A: the applied contract will be the API contract (using the example data)
 - For APP 2 and API A: the applied contract will be the group contract because it is the only contract between the application and the API (APP 2 does not have a contract made directly to the API A).
So that is why `applied` tag cannot be seen in the group instance contracts page because maybe for some APIs is the applied one but maybe for another, it is not (like the example)

Reply:
Done! Thanks!

Reply:
Hi @priyaayyar, could you point me out where the applied contract section was added? I can't find it

Reply:
Done! Thanks!